dotnet/src/webdriver/PinnedScript.cs

```diff
@@ -17,83 +17,70 @@
 // under the License.
 // </copyright>
 
-using System;
 using System.Globalization;
 
+#nullable enable
+
 namespace OpenQA.Selenium
 {
     /// <summary>
     /// A class representing a pinned JavaScript function that can be repeatedly called
     /// without sending the entire script across the wire for every execution.
     /// </summary>
-    public class PinnedScript
+    public sealed class PinnedScript
```

Comment:
Risky, let's don't touch it.

Reply:
Hm, no public ctor, then is it safe?

Reply:
Since we have an `internal` constructor, users cannot subtype this class anyway. There's a comment on the constructor explaining that this was intentional